frontend/app/src/plugins/rui/index.ts

```diff
@@ -443,6 +264,165 @@ export function createRuiPlugin(defaults: Partial<RuiOptions['defaults']>): RuiP
+        LuArrowUp,
+        LuCircleAlert,
+        LuCheck,
+        LuCirclePlus,
+        LuKey,
+        LuSettings,
+        LuPencilLine,
+        LuCable,
+        LuLockKeyhole,
+        LuArrowDownAZ,
+        LuArrowDownZA,
+        LuFileX,
+        LuSiren,
+        LuAlarmClock,
+        LuDownload,
+        LuUpload,
+        LuFlame,
+        LuShuffle,
+        LuListPlus,
+        LuCloud,
+        LuCloudOff,
+        LuLaptopMinimal,
+        LuKeyboard,
+        LuWifi,
+        LuWifiOff,
+        LuSave,
+        LuFilter,
+        LuInfo,
+        LuLeaf,
+        LuRocket,
+        LuOctagonAlert,
+        LuCombine,
+        LuTrophy,
+        LuEarth,
+        LuLink,
+        LuLink2,
+        LuEllipsisVertical,
+        LuFeather,
+        LuFolder,
+        LuSparkles,
+        LuBlocks,
+        LuSlidersHorizontal,
+        LuSkull,
+        LuGitBranch,
+        LuBug,
+        LuTriangleAlert,
+        LuLayoutGrid,
+        LuLockKeyholeOpen,
+        LuLayoutList,
+        LuServer,
+        LuFolderOpen,
+        LuMessageSquareQuote,
+        LuFileText,
+        LuGitCommitHorizontal,
+        LuHouse,
+        LuBox,
+        LuMedal,
+        LuBriefcase,
+        LuLuggage,
+        LuListTodo,
+        LuCoinsExchange,
+        LuSprout,
+        LuScrollText,
+        LuList,
+        LuFileSpreadsheet,
+        LuUserRoundCog,
+        LuBolt,
+        LuBuilding2,
+        LuCircleDollarSign,
+        LuPin,
+        LuPinOff,
+        LuSquarePen,
+        LuNonFungibleBalances,
+        LuScale,
+        LuEqual,
+        LuReceiptCent,
+        LuMapPinCheckInside,
+        LuHandshake,
+        LuPercent,
+        LuSquareKanban,
+        LuRepeat,
+        LuLogOut,
+        LuLandmark,
+        LuFileDown,
+        LuFileUp,
+        LuHeartHandshake,
+        LuBookText,
+        LuFileCog,
+        LuHandHeart,
+        LuHandCoins,
+        LuCircleArrowOutUpRight,
+        LuCornerDownLeft,
+        LuWalletMinimal,
+        LuHistory,
+        LuCornerUpLeft,
+        LuExternalLink,
+        LuDroplets,
+        LuArrowUpDown,
+        LuFileClock,
+        LuReceiptCent,
+        LuBadgeDollarSign,
+        LuSendToBack,
+        LuLayersIn,
+        LuLayersOut,
+        LuCloudSync,
+        LuCloudUpload,
+        LuCloudUpload2,
+        LuCloudDownload,
+        LuCloudDownload2,
+        LuRefund,
+        LuDeposits,
+        LuLiabilities,
+        LuDropletFill,
+        LuDropletHalfFill,
+        LuMoon,
+        LuSun,
+        LuPalette,
+        LuCornerLeftUp,
+        // remaining from prev version of premium components
+        RiEqualLine,
+        RiMoonLine,
+        RiSunLine,
+        RiArrowDownSLine,
+        RiRestartLine,
+        RiRefreshLine,
+        RiAddLine,
+        RiSubtractLine,
+        RiSendPlane2Line,
+        RiPaletteLine,
+        RiExternalLinkLine,
+        RiCornerLeftUpLine,
+        RiArrowUpCircleLine,
+        RiInformationLine,
```

Comment:
Ok so for this we need a breaking alpha release before we remove otherwise components will be broken in the meanwhile.